api-spec/transactions-api.yaml

```diff
@@ -436,18 +439,23 @@ components:
         holderName:
           type: string
           description: The card holder name
+        brand:
```

Comment:
This field is already modeled in `CardAuthRequestDetails`

Reply:
I didn't find it in the dto definition, so that I added it here in the definition of the dto itself.